src/nvim/api/ui.c

```diff
@@ -706,19 +706,21 @@ void redraw(uint64_t channel_id, Array uidata, Error *error)
   Array call = ARRAY_DICT_INIT;
   char *method_name;
   size_t size;
+  Error err = ERROR_INIT; 
 
   Array uidata_copy = copy_array(uidata);
+  error = &err;
```

Comment:
Don't "return" pointer to local variable from a function. After leaving the function the local variable does not exist anymore.